src/loaders/svg/tvgSvgCssStyle.cpp

```diff
@@ -32,13 +32,18 @@ static void _copyStyle(SvgStyleProperty* to, const SvgStyleProperty* from)
 {
     if (from == nullptr) return;
     //Copy the properties of 'from' only if they were explicitly set (not the default ones).
-    if (from->curColorSet && !(to->flags & SvgStyleFlags::Color)) {
+    if ((from->curColorSet && !(to->flags & SvgStyleFlags::Color)) ||
+        ((from->flagsImportance & SvgStyleFlags::Color) && !(to->flagsImportance & SvgStyleFlags::Color))) {
         to->color = from->color;
         to->curColorSet = true;
         to->flags = (to->flags | SvgStyleFlags::Color);
+        if (from->flagsImportance & SvgStyleFlags::Color) {
+            to->flagsImportance = (to->flagsImportance | SvgStyleFlags::Color);
+        }
     }
```

Comment:
When I searched for this rule, it seems that this rule is a very underused rule.
If you make it a function and reduce the code, the management would be better.

```cpp
bool _checkImportace(SvgStyleProperty* to, const SvgStyleProperty* from, SvgStyleFlags flag)   
{ 
   if (((from->flagsImportance & flag) && !(to->flagsImportance & flag))) {    
     if (from->flagsImportance & flag) {                                   
         to->flagsImportance = (to->flagsImportance | flag);
     }                                                      
     return true;
   }             
   return false;
}               


static void _copyStyle(SvgStyleProperty* to, const SvgStyleProperty* from)
{                                                                         
    if (from == nullptr) return;
    //Copy the properties of 'from' only if they were explicitly set (not the default ones).
    if (_checkImportace(to, from, SvgStyleFlags::Color) || 
        (from->curColorSet && !(to->flags & SvgStyleFlags::Color))    ) {           
        to->color = from->color;                          
        to->curColorSet = true; 
        to->flags = (to->flags | SvgStyleFlags::Color);
    }                                                  

```

Reply:
@JSUYA I added the _checkImportance function, but only with the checking part. setting has to be done in the original place, since there are two conditions that may let use into the 'if' and no matter which one is true, this has to be done: ``` if (from->flagsImportance & flag) to->flagsImportance = (to->flagsImportance | flag);```